apps/web/vitest.config.setup.ts

```diff
@@ -0,0 +1,105 @@
+import { createI18n } from 'vue-i18n';
+import { config } from '@vue/test-utils';
+
+const FindTestIdPlugin = (wrapper: any) => {
+  function findByTestId(testId: string) {
+    return wrapper.find(`[data-testid='${testId}']`);
+  }
+
+  return {
+    findByTestId,
+  };
+};
+
+const GetTestIdPlugin = (wrapper: any) => {
+  function getByTestId(testId: string) {
+    return wrapper.get(`[data-testid='${testId}']`);
+  }
+
+  return {
+    getByTestId,
+  };
+};
+
+config.plugins.VueWrapper.install(FindTestIdPlugin);
+config.plugins.VueWrapper.install(GetTestIdPlugin);
+
+const i18n = createI18n({
+  legacy: false,
+  fallbackWarn: false,
+  missing: (_locale, key) => key,
+});
+config.global.plugins = [i18n];
+config.global.mocks = {
+  $t: (key: string) => key,
+  $d: (date: Date) => date.toLocaleDateString(),
+};
+config.global.stubs = {
+  RouterLink: true,
+  NuxtImg: true,
+  'i18n-t': true,
+};
+
+import { mockComponent, mockNuxtImport } from '@nuxt/test-utils/runtime';
+import { CategoryTreeMock } from './__tests__/fixtures/category-tree.mock';
+import { ShippingCountriesMock } from './__tests__/fixtures/shipping-countries.mock';
+
+mockComponent('./app.vue', {
+  template: `
+  <Body class="font-body" :class="bodyClass" />
+  <VitePwaManifest />
+  <NuxtLayout>
+    <NuxtPage />
+  </NuxtLayout>`,
+});
+
+mockComponent('ClientOnly', {
+  template: `<div> <slot/> </div>`,
+});
+
+mockNuxtImport('useCategoryTree', () => {
+  return () => {
+    return { data: computed(() => CategoryTreeMock) };
+  };
+});
+
+mockNuxtImport('useActiveShippingCountries', () => {
+  return () => {
+    return {
+      data: computed(() => ShippingCountriesMock.data),
+      getActiveShippingCountries: () => {
+        return vi.fn().mockResolvedValue(ShippingCountriesMock.data);
+      },
+    };
+  };
+});
```

Comment:
**question**: Where do we need this mock? Can we just use it in the tests that actually interact with shipping countries?

Reply:
That's the problem, we can't use the mockNuxtImport in the test where we need it. This needs more research.

Reply:
Definitely. Maybe we can figure out a difference between our cases and the [nuxt/utils examples](https://github.com/search?q=repo%3Anuxt%2Ftest-utils+mocknuxtimport+path%3A%2F%5Eexamples%5C%2F%2F&type=code), if you haven't already looked at those.